python/delta/tables.py

```diff
@@ -1285,6 +1285,43 @@ def partitionedBy(
         ))
         return self
 
+    @overload
+    def clusteredBy(
+        self, *cols: str
+    ) -> "DeltaTableBuilder":
+        ...
+
+    @overload
+    def clusteredBy(
+        self, __cols: Union[List[str], Tuple[str, ...]]
+    ) -> "DeltaTableBuilder":
+        ...
+
+    @since(3.2)  # type: ignore[arg-type]
+    def clusteredBy(
```

Comment:
Could you rename this to `clusterBy` so that it is more inlined with sql `CLUSTER BY` API? For both scala and python.

Reply:
Sure, I just tried to match `partitionedBy`